CombinatorialGames/Game/Basic.lean

```diff
@@ -68,6 +68,7 @@ def ofSets (s t : Set Game.{u}) [Small.{u} s] [Small.{u} t] : Game.{u} :=
 
 @[inherit_doc] notation "{" s " | " t "}ᴳ" => ofSets s t
 
+@[simp]
```

Comment:
I'm uncomftorable with this being a `simp` tagged theorem, though this could be because I have a limited understanding of when this applies.

Would this apply more eagerly than congruence where `mk {s | t}ᴵ = mk {l | r}ᴵ iff {s | t}ᴵ = {l | r}ᴵ`?

Reply:
I think it's somewhat unlikely that we'd have a `simp` lemma of the form `{s | t}ᴵ = {l | r}ᴵ`. In the case where `s = l` and `t = r`, the presence of this lemma shouldn't prevent `simp` from closing the goal.